display_list/display_list.h

```diff
@@ -237,7 +237,9 @@ class DisplayListStorage {
   DisplayListStorage() = default;
   DisplayListStorage(DisplayListStorage&&) = default;
 
-  uint8_t* get() const { return ptr_.get(); }
+  uint8_t* get() { return ptr_.get(); }
+
+  const uint8_t* get() const { return ptr_.get(); }
```

Comment:
This is wrong. DisplayListStorage is *not* a const storage. It is treated as const in DisplayList where the field is declared const. That const on the field definition should suffice to make the non-const version of this getter not accessible within this file.

But DisplayListBuilder also uses this object and it absolutely is editing the memory held by the Storage object. Making this object provide only const pointers to the data is not really compatible with the way it is used in DisplayListBuilder. I'm not sure how this compiled, but it is likely because we are compiling the engine with a relaxed ability to cast away const modifiers (which means using this method on the object returned from the public accessor is also not really safe).

So, there is some illusion of const safety here, but it must be broken or inadequate or the DisplayListBuilder *should* have failed to compile...?